pkg/controller/spd/indicator-plugin/manager.go

```diff
@@ -39,6 +40,8 @@ type IndicatorUpdater interface {
 	AddBusinessIndicatorSpec(_ types.NamespacedName, _ []apiworkload.ServiceBusinessIndicatorSpec)
 	AddSystemIndicatorSpec(_ types.NamespacedName, _ []apiworkload.ServiceSystemIndicatorSpec)
 	AddBusinessIndicatorStatus(_ types.NamespacedName, _ []apiworkload.ServiceBusinessIndicatorStatus)
+
+	UpdateBaselineRatio(_ types.NamespacedName, _ float32)
```

Comment:
make the naming aligned, either by change the origin `AddBusinessIndicatorSpec` to `Updatexxx` or `UpdateBaselineRatio` to `AddBaselineRatio`